hack/install.sh

```diff
@@ -425,7 +425,7 @@ do_install() {
 			(
 			set -x
 			for key_server in $key_servers ; do
-				$sh_c "apt-key adv --keyserver hkp://${key_server}:80 --recv-keys ${gpg_fingerprint}" && break
+				$sh_c "apt-key adv --keyserver hkp://${key_server}:80 --keyserver-options http-proxy=$http_proxy --recv-keys ${gpg_fingerprint}" && break
```

Comment:
Is it a problem that this is added unconditionally? Should we add it *only* if a `$http_proxy` or `$HTTP_PROXY` is set?